backend/src/main/java/com/github/hamideh6182/model/Checkout.java

```diff
@@ -0,0 +1,15 @@
+package com.github.hamideh6182.model;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document("checkout")
+public record Checkout(
+        @Id
+        String id,
+        String bookId,
+        String checkoutDate,
+        String returnDate,
+        String userId
+) {
```

Comment:
maybe it would be good to add the name of the book to checkout record, so you can see it in mongoDB, when you need it